readme.md

```diff
@@ -134,6 +135,32 @@ Directories are recursive by default. Directories named `fixtures` and `helpers`
 
 When using `npm test`, you can pass positional arguments directly `npm test test2.js`, but flags needs to be passed like `npm test -- --verbose`.
 
+### The `--match` flag
```

Comment:
Hm. How's about `Matched-tests`? Too succinct? 

Thought it'd look better to keep in line with the other 'test scope' modifiers; 
`Serial-tests`, `Only-tests`, `Skip-tests`, `Todo-tests`